R/makephyloseqFromTreeSummarizedExperiment.R

```diff
@@ -0,0 +1,113 @@
+#' Create a phyloseq object from a TreeSummarizedExperiment object
+#'
```

Comment:
I know its quite redundant, but a description is needed.

Reply:
Added